pkg/apis/cstor/v1/cstorcompletebackup.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +resource:path=cstorcompletedbackup
+
+// CStorCompletedBackup describes a cstor completed-backup resource created as custom resource
+type CStorCompletedBackup struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              CStorCompletedBackupSpec `json:"spec"`
+}
+
+// CStorCompletedBackupSpec is the spec for a CStorBackup resource
+type CStorCompletedBackupSpec struct {
+	// BackupName is a name of the backup or scheduled backup
+	BackupName string `json:"backupName"`
+
+	// VolumeName is a name of the volume for which this backup is destined
+	VolumeName string `json:"volumeName"`
+
+	// SecondLastSnapName is a name of the second last completed-backup's snapshot
+	SecondLastSnapName string `json:"secondLastSnapName"`
```

Comment:
how about saying as 'SnapName' and 'PrevSnapName'. Sorry for picking on names

Reply:
basically, LastSnapName and SecondLastSnapName of a schedule keeps changing. So, thought that - its better to change them.

Reply:
@vishnuitta in CStorBackupSpec we have changed from  `SnapName and PrevSnapName` to `SecondLastSnapName and LastSnapName`. Should we still need to change their names?

Reply:
cc: @mynktl

Reply:
I don't see the change in CStorBackupSpec.

Reply:
Just a minute let me go back and look

Reply:
> I don't see the change in CStorBackupSpec.

Sorry for creating confusion and you are correct vitta we didn't change any CStorBackupSpec.

Reply:
@vishnuitta Existing name `snapName` and `PrevSnapName` were not conveying the information of snapshot sequence, due to that We planned to change it to `LastSnapName` and `SecondLastSnapName`.

Reply:
ok.. as this CR is not exposed to user, we should be good I guess.. otherwise, LastSnapName and SecondLastSnapName also doesn't convey the meaning..